### ISSUE TITLE
Add edit/delete functionality to threat dashboard option menu items

### ISSUE THREAD

**Issue by webiator** [unfetter-discover/unfetter#667]
The threat dashboard was modified in issue #637 to follow the ADG wireframes with the addition of the "generic" sidebar. The options menu items were left disabled until it could be determined what they were to be used for.

Edit - edit the currently displayed work product

Share - "share" the work product; not to be implemented yet, as sharing is not a current concept

Delete - delete the work product

Basically, the Edit and Delete actions are the same what the threat overview page provided, so copying over these actions to the threat dashboard will fulfill this requirement.
